var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -73,6 +73,13 @@ class Trilinos(CMakePackage):
             description='global ordinal type for Tpetra')
     variant('fortran',      default=True,
             description='Compile with Fortran support')
+    variant('cuda', default=sys.platform != 'darwin',
+            description='Build with CUDA support')
```

Comment:
I don't think `cuda` should default to on (on systems that support it), especially since CUDA support is available for only a small fraction of trilinos packages.

Reply:
Doesn't the cuda variant get inherited from CudaPackage anyway?

Reply:
Yeah, the difference is that by overriding it, a package can specify it to be *on* by default (or in this case, on when building on non-mac systems).